### ISSUE TITLE
Review settings for publishing snapshots

### ISSUE THREAD

**Issue by renatocaval** [akka/akka-projection#37]
`master` is there to do the snapshot releases on every merge to master.

_Originally posted by @ennru in https://github.com/akka/akka-projection/pull/30_

**Comment by ihostage** [comment]
How about using Travis Deploy with the condition for publishing? 
https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on

> _Note_: that pull request builds skip the deployment step altogether.

Something like this:
```
deploy:
  - provider: script
    skip_cleanup: true
    script: deploy.sh
    on:
      ...
```

**Comment by renatocaval** [comment]
@ihostage, we can consider it on a later phase. For the moment, I prefer to keep the same we are doing on other builds.

**Comment by ennru** [comment]
Fixed with #39
